tests/suites/test_suite_ecp.function

```diff
@@ -1433,3 +1433,69 @@ exit:
     mbedtls_free(p);
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS */
+void ecp_mod_mul_inv(char *input_A, int id, int ctype, int iret)
+{
+    int ret;
+    mbedtls_mpi_mod_modulus m;
+    mbedtls_mpi_mod_modulus_init(&m);
+    mbedtls_mpi_mod_residue A = { NULL, 0 };
+    mbedtls_mpi_mod_residue A_inverse = { NULL, 0 };
+    mbedtls_mpi_mod_residue rX = { NULL, 0 };
+    mbedtls_mpi_uint *rX_raw = NULL;
+    mbedtls_mpi_uint *p_inverse = NULL;
+    mbedtls_mpi_uint *p = NULL;
+    mbedtls_mpi_uint *bufx = NULL;
+    mbedtls_mpi_uint Result = 1;
+
+    ret = mbedtls_ecp_modulus_setup(&m, id, ctype);
+    TEST_EQUAL(ret, iret);
+
+    if (ret == 0) {
+
+        size_t limbs;
+
+        TEST_EQUAL(mbedtls_test_read_mpi_core(&p, &limbs, input_A), 0);
+        TEST_EQUAL(0, mbedtls_mpi_mod_residue_setup(&A, &m, p, limbs));
+
+        /* Test for limb sizes */
+        TEST_EQUAL(m.limbs, limbs);
+
+        /* Test for validity of moduli by the presence of Montgomery consts */
+
+        TEST_ASSERT(m.rep.mont.mm != 0);
+        TEST_ASSERT(m.rep.mont.rr != NULL);
+
+        ASSERT_ALLOC(p_inverse, limbs);
+        TEST_EQUAL(0, mbedtls_mpi_mod_residue_setup(&A_inverse, &m, p_inverse, limbs));
+
+        ASSERT_ALLOC(rX_raw, limbs);
+        TEST_EQUAL(0, mbedtls_mpi_mod_residue_setup(&rX, &m, rX_raw, limbs));
+
+        /* Get inverse of A mode m, and multiply it with itself,
+         * to see whether the result equal to '1' */
+        TEST_EQUAL(0, mbedtls_mpi_mod_inv(&A_inverse, &A, &m));
+        TEST_EQUAL(mbedtls_mpi_mod_mul(&rX, &A, &A_inverse, &m), 0);
+
+        ASSERT_ALLOC(bufx, limbs);
+        TEST_EQUAL(mbedtls_mpi_mod_write(&rX, &m, (unsigned char *)bufx,
```

Comment:
I think the code style needs that to be:

TEST_EQUAL(mbedtls_mpi_mod_write(&rX, &m, (unsigned char *)bufx,
```suggestion
        TEST_EQUAL(mbedtls_mpi_mod_write(&rX, &m, (unsigned char *) bufx,
```